.gitignore

```diff
@@ -24,3 +24,5 @@ bin
 *~
 
 hack/*.kubeconfig
+
+alice-oil.kubeconfig
```

Comment:
hum, why `alice-oil.kubeconfig` here? It should be generated inside the `hack` folder, and so ignored.

Reply:
In the script there is:
`cat > ${USER}-${TENANT}.kubeconfig`
If you execute it from the main folder with:
`./hack/create-user.sh alice oil`
You get the file in the main folder.

In case I could adapt it in a additional PR. 

WDYT?

Reply:
Oh, it makes sense: let's __gitignore__ `*.kubeconfig` everywhere.

```
sed -ie 's#hack/*.kubeconfig#*.kubeconfig#'
```

Reply:
done.